### DISCUSSION TITLE
Custom Empire and System Effects

### DISCUSSION THREAD

**Discussion by Clashsoft** [sekassel/stp-24-server-issues#15] [General]
Please comment your use of the `Empire` and `System` custom `effects`.

| ID | Scope | Condition | Effects | Duration |
|--------|--------|--------|--------|--------|
| Team S |
| `abundance` | Empire | MTTH: 100 periods | +100% minerals <br/> +15 alloys pro mining site | 25 periods |
| `black_spot` | Empire | MTTH: 100 periods | -10 alloys <br/> -20 fuel pro 100 credits | 10 periods |
| `crapulence` | Empire | MTTH: 100 periods | +25% food <br/> +25 consumer_goods | 20 periods |
| `dutchman` | Empire | MTTH: 100 periods |  -20 fuel <br/> -50 credits | 25 periods |
| `equiv_ex` | Empire | MTTH: 100 periods |  +0.5 alloys pro fuel <br/> +0.5 fuel pro alloys | 5 periods |
| `fools_gold` | Empire | MTTH: 100 periods | -20 credits <br/> -10 alloys <br/> +30 fuel  | 15 periods |
| `grand_exp` | Empire | MTTH: 100 periods | +5 research pro owned systems by other empires | 20 periods |
| `pestilence` | Empire | MTTH: 100 periods | -20 population <br/> +15 energy <br/> -50% food | 15 periods |
| `reckoning` | Empire | MTTH: 100 periods | +5 population <br/> +1 credits pro population | 10 periods |
| `roger_feast` | Empire | MTTH: 100 periods | +5% population <br/> +5% credits <br/> +5% food<br/> +5% consumer_goods | 30 periods |
| `rum_bottle` | Empire | MTTH: 100 periods | -5 research pro consumer_goods <br/> -0.5% research pro consumer_goods | 20 periods |
| `submerge` | Empire | MTTH: 100 periods | -50% for every production unit | 20 periods |
| Team R |
| covid19 | Empire | MTTH: 120 periods | system fuel production -50% | 18  periods |
| mailStart | Empire | MTTH: 100 periods | research -10 | - |
| mailEsc1 | Empire | - | -100 Research | - |
| mailSecured | Empire | - | energy consumption -5% | - |
| blunderStart | System | MTTH: 100 periods | -50% production for every building | 24 |
| bunderBad | System | - | -100% exchange production, -10% population growth | 12 |
| blunderNeutral | Empire | - | +100 Credits | - |
| blunderGood | Empire | - | +100 Credits | - |
| catStart | Empire | MTTH: 80 periods | +2% population growth | 6 |
| exchangeChaos | Empire | - | -10% exchange production | 3 |
| refineryChaos | Empire | - | -10% refinery production | 3 |
| catCaught | Empire | - | +10 technology | - |
| catEnd | Empire | - | +10 technology +50 fuel | - |
| Team P |
| `blizzard` | Empire | MTTH: 400 periods | -50 energy every period <br/> -50 fuel every period | 15 periods |
| `crop_failure` | System | MTTH: 400 periods | Agriculture food production: -100% <br/> Farm food production: -100% | 15 periods |
| `earth_quake` | Empire | MTTH: 500 periods | Farm food production: -90% <br/> Refinery fuel production: -90% <br/> Research lab research production: -90% <br/> Mine minerals production: -90% <br/> Power plant energy production: -90% <br/> Exchange credits production: -90% <br/> -40 credits every period | 15 periods |
| `inflation` | Empire | MTTH: 350 periods | Market fee: +50% | 15 periods |
| `thirst_for_knowledge` | Empire | MTTH: 500 periods | -50 credits every period <br/> Every technology cost: -20% | 30 periods |
| `scientific_breakthrough` | Empire | MTTH: 400 periods | +50 research every period | 20 periods |
| `treasure` | Empire | MTTH: 350 periods | +60 credits every period | 5 periods |
| `resignation.physics` | Empire | MTTH: 500 periods | Changes the scientist in the category "Physics" | - |
| `resignation.society` | Empire | MTTH: 500 periods | Changes the scientist in the category "Society" | - |
| `resignation.engineering` | Empire | MTTH: 500 periods | Changes the scientist in the category "Engineering" | - |
| `mysterious_liquid` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Industry consumer goods production: +50% <br/> Ancient factory consumer goods production: +50% <br/> Factory consumer goods production: +50% | 20 periods |
| `mysterious_liquid` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Industry consumer goods production: -100% <br/> Ancient factory consumer goods production: -100% <br/> Factory consumer goods production: -100% | 20 periods |
| `group` | Empire | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | +30 credits every period <br/> +30 minerals every period <br/> +30 alloys every period <br/> +30 food every period | 20 periods |
| `group` | Empire | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | -30 credits every period <br/> -30 minerals every period <br/> -30 alloys every period <br/> -30 food every period | 20 periods |
| `marriage` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Every system upgrade cost: -20% | 20 periods |
| `marriage` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Every system upgrade cost: +20% | 20 periods |

\* - MTTH: [Mean Time to Happen](https://stellaris.paradoxwikis.com/Event_modding#Mean_Time_To_Happen)

Please use the following format.
Template for copy & paste:

```md
| ID | Scope | Condition | Effects | Duration |
|--------|--------|--------|--------|--------|
| `gold_rush` | System | MTTH: 100 periods | mining districts +25% | 10 periods |
| `famine` | Empire | food < 100 | all resources -15% <br/> population growth -5% | until food > 100 |
```

You can use `<br/>` within cells to add line breaks.

**Comment by AshkanKiafard** [comment]
All of the effects below are for the empire

| ID | Condition | Effects | Duration |
|--------|--------|--------|--------|
| `abundance` | MTTH: 100 periods | +100% minerals <p> +15 alloys pro mining site | 25 periods |
| `black_spot` | MTTH: 100 periods | -10 alloys <p> -20 fuel pro 100 credits | 10 periods |
| `crapulence` | MTTH: 100 periods | +25% food <p> +25 consumer_goods | 20 periods |
| `dutchman` | MTTH: 100 periods |  -20 fuel <p> -50 credits | 25 periods |
| `equiv_ex` | MTTH: 100 periods |  +0.5 alloys pro fuel <p> +0.5 fuel pro alloys | 5 periods |
| `fools_gold` | MTTH: 100 periods | -20 credits <p> -10 alloys <br> +30 fuel  | 15 periods |
| `grand_exp` | MTTH: 100 periods | +5 research pro owned systems by other empires | 20 periods |
| `pestilence` | MTTH: 100 periods | -20 population <p> +15 energy <br> -50% food | 15 periods |
| `reckoning` | MTTH: 100 periods | +5 population <p> +1 credits pro population | 10 periods |
| `roger_feast` | MTTH: 100 periods | +5% population <br> +5% credits<br> +5% food<br> +5% consumer_goods | 30 periods |
| `rum_bottle` | MTTH: 100 periods | -5 research pro consumer_goods<br> -0.5% research pro consumer_goods | 20 periods |
| `submerge` | MTTH: 100 periods | -50% for every production unit | 20 periods |

**Reply by Clashsoft** [reply]
Thanks, please specify the scope for each effect (Empire or System).

**Reply by AshkanKiafard** [reply]
> Thanks, please specify the scope for each effect (Empire or System).

I just saw your comment.
All of them are for the empire.
I edited my original comment.

**Comment by 35529110** [comment]
ID | Scope | Condition | Effects | Duration |
|--------|--------|--------|--------|--------|
| covid19 | Empire | MTTH: 120 periods | system fuel production -50% | 18  periods |
| mailStart | Empire | MTTH: 100 periods | research -10 | - |
| mailEsc1 | Empire | - | -100 Research | - |
| mailSecured | Empire | - | energy consumption -5% | - |
| blunderStart | System | MTTH: 100 periods | -50% production for every building | 24 |
| bunderBad | System | - | -100% exchange production, -10% population growth | 12 |
| blunderNeutral | Empire | - | +100 Credits | - |
| blunderGood | Empire | - | +100 Credits | - |
| catStart | Empire | MTTH: 80 periods | +2% population growth | 6 |
| exchangeChaos | Empire | - | -10% exchange production | 3 |
| refineryChaos | Empire | - | -10% refinery production | 3 |
| catCaught | Empire | - | +10 technology | - |
| catEnd | Empire | - | +10 technology +50 fuel | - |

**Reply by Clashsoft** [reply]
Please report your other effects here as well. This discussion is not about the events as the server does not care about them. It's just about the effects.

Note that 900 periods is currently way longer than anyone would play the game for. Please adjust so there is a chance to encounter the event.

**Reply by 35529110** [reply]
I updated my comment to include all our needed effects and adjusted the periods accordingly. 
Please note, that effects without a condition are meant to only be able to be added manually

**Comment by wbrrobin** [comment]
| ID | Scope | Condition | Effects | Duration |
|--------|--------|--------|--------|--------|
| `blizzard` | Empire | MTTH: 400 periods | -50 energy every period <br/> -50 fuel every period | 15 periods |
| `crop_failure` | System | MTTH: 400 periods | Agriculture food production: -100% <br/> Farm food production: -100% | 15 periods |
| `earth_quake` | Empire | MTTH: 500 periods | Farm food production: -90% <br/> Refinery fuel production: -90% <br/> Research lab research production: -90% <br/> Mine minerals production: -90% <br/> Power plant energy production: -90% <br/> Exchange credits production: -90% <br/> -40 credits every period | 15 periods |
| `inflation` | Empire | MTTH: 350 periods | Market fee: +50% | 15 periods |
| `thirst_for_knowledge` | Empire | MTTH: 500 periods | -50 credits every period <br/> Every technology cost: -20% | 30 periods |
| `scientific_breakthrough` | Empire | MTTH: 400 periods | +50 research every period | 20 periods |
| `treasure` | Empire | MTTH: 350 periods | +60 credits every period | 5 periods |
| `resignation.physics` | Empire | MTTH: 500 periods | Changes the scientist in the category "Physics" | - |
| `resignation.society` | Empire | MTTH: 500 periods | Changes the scientist in the category "Society" | - |
| `resignation.engineering` | Empire | MTTH: 500 periods | Changes the scientist in the category "Engineering" | - |
| `mysterious_liquid` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Industry consumer goods production: +50% <br/> Ancient factory consumer goods production: +50% <br/> Factory consumer goods production: +50% | 20 periods |
| `mysterious_liquid` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Industry consumer goods production: -100% <br/> Ancient factory consumer goods production: -100% <br/> Factory consumer goods production: -100% | 20 periods |
| `group` | Empire | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | +30 credits every period <br/> +30 minerals every period <br/> +30 alloys every period <br/> +30 food every period | 20 periods |
| `group` | Empire | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | -30 credits every period <br/> -30 minerals every period <br/> -30 alloys every period <br/> -30 food every period | 20 periods |
| `marriage` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Every system upgrade cost: -20% | 20 periods |
| `marriage` | System | MTTH: 300 periods <br/> User answered "yes" to question <br/> 50% chance | Every system upgrade cost: +20% | 20 periods |

**Reply by Clashsoft** [reply]
Please adjust your MTTHs to be in the range of 50-200 periods. 500 is too long to happen in our games.
